src/applications/personalization/view-dependents/components/ViewDependentsList/ViewDependentsList.jsx

```diff
@@ -1,39 +1,41 @@
 import React, { Component } from 'react';
-import ViewDependentsListItem from './ViewDependentsListItem';
+import PropTypes from 'prop-types';
+import ViewDependentsListItem from '../ViewDependentsList/ViewDependentsListItem';
 
 class ViewDependentsList extends Component {
   render() {
-    return (
-      <>
-        <h3>Dependents on award</h3>
-        <p className="vads-u-padding-bottom--2">
-          Dependents on award have been added to you disability claim.{' '}
-          <strong>
-            If a dependents status has changed, you need to let the VA know.
-          </strong>
-        </p>
-        <ViewDependentsListItem
-          onAward
-          social="312-314-1415"
-          birthdate="05/05/1982"
-          age={37}
-        />
-        <h3>Dependents not on award</h3>
-        <p>
-          Dependents not on award may be awaiting a decision, or they were once
-          added to your disability claim and their status has changed.{' '}
-          <strong>Let the VA know if a dependent's status has changed</strong>{' '}
-          and they are now eligible to be added to your disability claim.
-        </p>
+    let mainContent;
+
+    if (this.props.dependents) {
+      mainContent = this.props.dependents.map((dependent, index) => (
         <ViewDependentsListItem
-          onAward
-          social="312-314-1415"
-          birthdate="05/05/1982"
-          age={37}
+          key={index}
+          name={dependent.name}
+          spouse={dependent.spouse}
+          onAward={dependent.onAward}
+          social={dependent.social}
+          birthdate={dependent.birthdate}
+          age={dependent.age}
         />
+      ));
+    } else {
+      mainContent = <p>No dependents in this list.</p>;
+    }
+
+    return (
+      <>
+        <h3>{this.props.header}</h3>
+        <p className="vads-u-padding-bottom--2">{this.props.subHeader}</p>
+        {mainContent}
       </>
```

Comment:
Not sure how much this actually matters, just something I noticed. Because fragments do not add any extra DOM nodes, this renders both lists in one `div`:
<img width="513" alt="Screen Shot 2019-12-11 at 3 37 40 PM" src="https://user-images.githubusercontent.com/15097156/70669955-ade7a680-1c2c-11ea-9c17-98daa065a3bd.png">

As opposed to:
<img width="528" alt="Screen Shot 2019-12-11 at 3 40 55 PM" src="https://user-images.githubusercontent.com/15097156/70669970-b9d36880-1c2c-11ea-87c3-858bc6fa6340.png">

Reply:
Great catch! Yeah I updated the list component to return in a div rather than a fragment